### PR TITLE
Fp16 msg

### PR DESCRIPTION
**Pull request by anj-s**
Couple of changes in this PR:

-Added an error message if users specify fp16 dtype when running Keras Resnet benchmarks.
-Cleaned up some print debug statements
-Added a formatted message that will print time taken to process x batches.
-Turned off ProgressBar Logger (should improve performance to some extent)

### CODE REVIEW

File: official/resnet/keras/keras_imagenet_main.py [hunk 1]
```diff
@@ -117,6 +117,13 @@ def run_imagenet_with_keras(flags_obj):
   Args:
     flags_obj: An object containing parsed flag values.
   """
+  # Raise an error if the user wants to run with FP16 support. Currently we do
```
Comment (tfboyd) [inline official/resnet/keras/keras_imagenet_main.py]: nit.  comment not needed, the error message is self explanatory.  The Error message could also be shorter with something like:  "fp16 not supported in Keras. Use fp32 (default)."   If I wish I was better with punctuation.

**Comment by anj-s** [comment]
Do you think it makes sense to say something like "dtype fp16 is not supported" just to tie the error message to the flag being used?

**Comment by tfboyd** [comment]
Yeah, I am marking approved and if you can slip in the dtype that would be great.

**Comment by tfboyd** [comment]
ahh I am bad at GitHub.  Merge the PR when you are ready.  If you are not already a collaborator I am adding you right now.

**Pull request merged**